.codesandbox/tasks.json

```diff
@@ -1,16 +1,4 @@
 {
-  // These tasks will run in order when initializing your CodeSandbox project.
-  "setupTasks": [
-    {
-      "name": "update pnpm",
-      "command": "npm install -g pnpm --force"
-    },
-    {
-      "name": "Install Dependencies",
```

Comment:
I think we'll still need the "Install Dependencies" step, but we can remove the "update pnpm" step!